common/components/Footer/index.jsx

```diff
@@ -94,102 +95,105 @@ const LINKS_SOCIAL = [
 export default class Footer extends Component {
   render() {
     return (
-      <footer className="Footer" role="contentinfo" aria-label="footer">
-        <div className="Footer-column Footer-about">
-          <p aria-hidden="true">
-            <a href="/">
-              <img
-                className="Footer-about-logo"
-                src={logo}
-                height="55px"
-                width="auto"
-                alt="MyEtherWallet"
-              />
-            </a>
-          </p>
-          <p className="Footer-about-text">
-            <span>
-              {translate('FOOTER_1')}
-            </span>
-            <span>
-              {translate('FOOTER_1b')}
-            </span>
-          </p>
+      <div>
+        <PreFooter />
+        <footer className="Footer" role="contentinfo" aria-label="footer">
```

Comment:
Most of this is noise that was introduced by an additional layer of nesting created by the top-level div.